From a3d7489199d71ab2129cf5274e31ee1a189a7a92 Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Thu, 31 Mar 2016 13:37:47 +0300
Subject: [PATCH 112/347] dt-binding: genpd: k2g: Add eDMA1 device id to the
 list

This patch comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

k2g have two eDMA controllers, add device definition for eDMA1

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
(cherry picked from commit 255617cf3ff0fbef61e92d6f376bc7b1b98017c8)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 include/dt-bindings/genpd/k2g.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/dt-bindings/genpd/k2g.h b/include/dt-bindings/genpd/k2g.h
index f8309e1..f8a365e 100644
--- a/include/dt-bindings/genpd/k2g.h
+++ b/include/dt-bindings/genpd/k2g.h
@@ -73,4 +73,5 @@
 #define K2G_DEV_MSMC0			0x0047
 #define K2G_DEV_CBASS0			0x0049
 #define K2G_DEV_BOARD0			0x004c
+#define K2G_DEV_EDMA1			0x004f
 #endif
-- 
1.7.5.4

